client/src/containers/routing/index.tsx

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
 import { Switch } from 'react-router-dom';
 import Example from 'containers/example';
-import { PublicRoute } from 'components';
+import { PublicRoute } from 'components/basic';
+import { SignInPage } from 'components/pages';
 
-interface IRoutingProps { }
+interface IRoutingProps {}
 
 const Routing: React.FC<IRoutingProps> = () => (
 	<Switch>
 		<PublicRoute exact restricted={false} path="/" component={Example} />
+		<PublicRoute exact restricted={false} path="/sign-in" component={SignInPage} />
```

Comment:
There should be `SignInPage` container. See how it works with Example.

So, you should have container which should return `SignInPage` component. Then we will add redux logic to container and component still will be reusable.

Also you should add callback props to SignInPage component. Like `onSubmitForm: (values: IFormValuesInterface) => void; onGitHubSignIn: () => void; etc...`. All logic must be in container